src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -5,20 +5,14 @@
 import edu.wpi.first.wpilibj.DigitalInput;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import frc.robot.constants.RobotConfig;
 import frc.robot.constants.RobotConstants.IntakeConstants;
-import frc.robot.constants.RobotConstants.ShooterConstants;
 
 public class Intake extends SubsystemBase {
-  private CANSparkMax m_shooterRollerMotor;
   private final CANSparkMax m_intakeRollerMotor; // Intake roller motor
   private final DigitalInput m_linebreak;
 
   /** Creates a new ExampleSubsystem. */
   public Intake() {
-    // Roller
-    m_shooterRollerMotor =
-        new CANSparkMax(ShooterConstants.kRollerMotorLeftId, MotorType.kBrushless);
     m_intakeRollerMotor = new CANSparkMax(IntakeConstants.kMotorID, MotorType.kBrushless);
     // TODO maybe use to terminate intake command
     m_linebreak = new DigitalInput(IntakeConstants.kLineBreakSensor);
```

Comment:
Should the line break sensor be in the indexer subsystem if it is mounted on that part of the robot?